### ISSUE TITLE
Removing underline no longer working...

### ISSUE THREAD

**Issue by mikeerickson** [enquirer/enquirer#97]
I originally raised this question here https://github.com/enquirer/enquirer/issues/81 and @doowb supplied a working solution.  However, that solution is no longer working (I feel it "broke" with the 2.2.0 release.

Could you please have a quick look at the code I supplied in the OP and see if you can reproduce.

**Comment by doowb** [comment]
@mikeerickson use `em` now instead of `heading`. `heading` doesn't really describe what the style was doing so it was changed to `em`:

```js
const { prompt } = require("enquirer");
const colors = require("ansi-colors");

const questions = {
  type: "multiselect",
  message: "Selecting item, no longer shows underline:",
  choices: ["item1", "item2", "item3"],
  styles: {
    // the underlined choices use `em` now since it's more semantic than `heading`
    em: colors.cyan
  },
  pointer(state, choice) {
    return choice.index === state.index ? colors.cyan.bold(colors.symbols.pointer) : " ";
  },
  indicator(state, choice) {
    return choice.enabled ? " " + colors.green("●") : " " + colors.gray("o");
  }
};

prompt(
  [questions],
  result => {
    console.log(result);
  },
  err => {
    console.error(err);
  }
);
```

We'll try to make sure these are described in the docs.

**Comment by mikeerickson** [comment]
Awesome thx 🤓

**Comment by jonschlinkert** [comment]
I just saw this! fwiw this was unintentionally committed by me. The `.heading()` method didn't seem to make any sense at all, as it's not semantic, so I played around with using `.em()`. My intention was to commit that in the 3.0 release, but clearly I accidentally forgot to remove it before pushing up 2.2. This was a breaking change and should not have been committed, but at this point, I think it would be better to wait and see if it affects anyone else before causing further confusion by reverting... 

My apologies!
